app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb

```diff
@@ -1,8 +1,8 @@
 <%=
   play_roles = if hostgroup.all_ansible_roles.empty?
-    "<a title='No Roles assigned' href=\"#\">#{_('Play Roles')}</a>".html_safe
+    "<a title='No Roles assigned' href=\"#\">#{_('Run all Ansible roles')}</a>".html_safe
```

Comment:
Not your fault here, but we have a button to `run ansible roles` which does nothing, shouldn't it be at least disabled? Or completely disappear - with just a placeholder message to inform user there are no roles? This seems bit bad UX. @ares, @domitea ?
The `disabled="disabled"` would be sufficient for me (we do it everywhere like that, but I think right UX is the button dissapears with an info)

Reply:
I think the button should stay so people don't get confused why sometimes it's missing. I agree disabling would be nice. If we decide to go that way, it should have title (appear on hover) explaining why it's disabled. @domitea is it ok to include this to this PR or should we track it separately? I believe it's a good improvement though and since we're touching this, it makes sense to do it.